src/components/app.js

```diff
@@ -39,15 +39,15 @@ render(
     <ConnectedRouter history={history}>
       <div>
         <Switch>
-          <Route path='/admin-login' component={AdminLogin} />
-          <Route path='/login' component={Login} />
-          <Route path='/logout' component={Logout} />
-          <Route path='/forgot_password/:token/' component={SetPassword} />
-          <Route path='/forgot_password' component={ForgotPassword} />
-          <Route path='/signup/:token' component={SignUp} />
-          <Route path='/oauth/callback' component={OAuthCallback} />
-
-          <Route path='/' component={Layout} />
+          <Route component={AdminLogin} path='/admin-login' />
```

Comment:
Here's one case where I find the non alphabetical ordering more meaningful from a code reading point of view.

But I can live with it.